index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"> <br>
+    <title>Odin</title>
+    
```

Comment:
⚠️ 
- Please note do not apply for a review if you haven't completed the project or still working on it